noxfile.py

```diff
@@ -123,9 +125,8 @@ def system(session):
     # Install all test dependencies, then install this package into the
     # virtualenv's dist-packages.
     session.install(
-        "mock", "pytest", "google-cloud-testutils",
+        "mock", "pytest", "google-cloud-testutils", ".[fastavro,pandas,pyarrow]"
     )
-    session.install("-e", ".[fastavro,pandas,pyarrow]")
     session.install("-e", ".")
```

Comment:
Why are we combining those "extras" install with the test deps installation.

Also, seems like the `session.install(-e, ".")` was redundant with the previous `session.install("-e", ".[fastavro,pandas,pyarrow]")` line.

Reply:
It's probably something in the template logic, I can get around this with a synth replacement rule.